src/sbt-test/sbt-plugins/simple/test

```diff
@@ -0,0 +1,29 @@
+# See http://eed3si9n.com/testing-sbt-plugins for sbt-scripted test syntax
+# initialize a git repository to make our plugins happy
```

Comment:
No you don't. sbt-scripted copies the test into a temp directory and runs the script there so anything you produce get's wiped when the test completes.